src/extension.ts

```diff
@@ -1411,7 +1411,7 @@ export class Ext extends Ecs.System<ExtEvent> {
 
                 /** Display an overlay indicating where the window will be placed if dropped */
 
-                if (overview.visible || !win || op !== 1) return
+                if (overview.visible || !win || is_keyboard_op(op) || is_resize_op(op)) return
```

Comment:
I've tried to replicate the semantics of the earlier code, but it's unclear to me what the condition here is. It doesn't seem to hurt to display overlays if we're doing a keyboard move op, but the previous code doesn't.